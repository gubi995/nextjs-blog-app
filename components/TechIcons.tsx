import React from 'react';
import { Avatar, styled } from '@material-ui/core';

const StyledTechIcons = styled('div')(({ theme }) => ({
  display: 'flex',
  flexWrap: 'wrap',
  justifyContent: 'space-evenly',
  width: '100%',
  margin: theme.spacing(5),
  '& > *': {
    margin: theme.spacing(2),
  },
}));

const TechIcons = () => {
  return (
    <StyledTechIcons>
      <Avatar variant="square" alt="html" src="/prog-icons/Html.png" />
      <Avatar variant="square" alt="css" src="/prog-icons/Css.png" />
      <Avatar variant="square" alt="js" src="/prog-icons/Javascript.png" />
      <Avatar variant="square" alt="ts" src="/prog-icons/Typescript.png" />
      <Avatar variant="square" alt="npm" src="/prog-icons/Npm.png" />
      <Avatar variant="square" alt="yarn" src="/prog-icons/Yarn.png" />
      <Avatar variant="square" alt="webpack" src="/prog-icons/Webpack.png" />
      <Avatar variant="square" alt="react" src="/prog-icons/React.png" />
      <Avatar variant="square" alt="angular" src="/prog-icons/Angular.png" />
      <Avatar variant="square" alt="node" src="/prog-icons/Nodejs.png" />
      <Avatar variant="square" alt="mongo" src="/prog-icons/Mongodb.png" />
    </StyledTechIcons>
  );
};

export default TechIcons;
