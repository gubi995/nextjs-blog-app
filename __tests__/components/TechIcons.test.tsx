import React from 'react';
import { render } from '@testing-library/react';

import TechIcons from '../../components/TechIcons';

describe('TechIcons', () => {
  test("should rendered component matches with it's the snapshot", () => {
    const { asFragment } = render(<TechIcons />);

    expect(asFragment()).toMatchSnapshot();
  });
});
