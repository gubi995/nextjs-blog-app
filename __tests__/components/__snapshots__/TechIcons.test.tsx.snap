// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TechIcons should rendered component matches with it's the snapshot 1`] = `
<DocumentFragment>
  <div
    class="div-root-1 div-root-2"
  >
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="html"
        class="MuiAvatar-img"
        src="/prog-icons/Html.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="css"
        class="MuiAvatar-img"
        src="/prog-icons/Css.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="js"
        class="MuiAvatar-img"
        src="/prog-icons/Javascript.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="ts"
        class="MuiAvatar-img"
        src="/prog-icons/Typescript.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="npm"
        class="MuiAvatar-img"
        src="/prog-icons/Npm.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="yarn"
        class="MuiAvatar-img"
        src="/prog-icons/Yarn.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="webpack"
        class="MuiAvatar-img"
        src="/prog-icons/Webpack.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="react"
        class="MuiAvatar-img"
        src="/prog-icons/React.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="angular"
        class="MuiAvatar-img"
        src="/prog-icons/Angular.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="node"
        class="MuiAvatar-img"
        src="/prog-icons/Nodejs.png"
      />
    </div>
    <div
      class="MuiAvatar-root MuiAvatar-square"
    >
      <img
        alt="mongo"
        class="MuiAvatar-img"
        src="/prog-icons/Mongodb.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
